src/azul/service/manifest_service.py

```diff
@@ -842,9 +834,8 @@ def _samples_tsv(self, bundle_tsv: IO[str]) -> None:
 
             # Extract fields from the sole inner file entity_type
             file = one(doc['contents']['files'])
-            file_cells = dict(file_url=self._dss_url(file))
+            file_cells = {}
```

Comment:
Can this variable (and its usage) be removed if it's no longer used?

Reply:
It is still used: https://github.com/DataBiosphere/azul/blob/34cc7167cc378e210d5c718699610756c64ba6aa/src/azul/service/manifest_service.py#L870

This is how all the file-specific columns are added to the manifest. The only change is that they are now all populated from the column configuration, instead of having "extra" ones.